remoulade/middleware/retries.py

```diff
@@ -86,8 +86,9 @@ def after_process_message(self, broker, message, *, result=None, exception=None)
             message.fail()
             return
 
-        message.options["retries"] += 1
-        message.options["traceback"] = traceback.format_exc(limit=30)
+        new_message = message.copy()
+        new_message.options["retries"] += 1
```

Comment:
we need a test of this beahvior

Reply:
This part is no longer necessary, do we still keep it ?